api/src/pcapi/core/search/backends/algolia.py

```diff
@@ -511,6 +511,7 @@ def serialize_venue(cls, venue: offerers_models.Venue) -> dict:
             "banner_url": venue.bannerUrl,
             "_geoloc": position(venue),
             "has_at_least_one_bookable_offer": has_at_least_one_bookable_offer,
+            "date_created": venue.dateCreated.timestamp(),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Pourquoi timestamp ici et pas un format de date lisible ?

Reply:
Vu ensemble: c'est cohérent avec les autres dates dans Alglia :+1: